File: JavaAssignment-2/swap.java
import java.util.*;
class swap
{
public static void main(String[] args)
{
int num1,num2,temp;
System.out.println("Enter 2 numbers to be swapped");
Scanner sc=new Scanner(System.in);
num1=sc.nextInt();
num2=sc.nextInt();
temp=num1;
num1=num2;
num2=temp;
System.out.println("----------------Displaying Swapped values\n"+"Value of first number: "+num1+"\n"+"Value of second number: "+num2);
}
}

File: C++ Assignments/Assignment 2/swap.cpp
#include<iostream>
using namespace std;
class swAp
{
int num1, num2, m, n;
public: void display() 
{
cout<<"Enter 2 numbers \n";
cin>>num1>>num2;
m=num1;
num1=num2;
num2=m;
cout<<"Swapped integers num1, num2 are\n"<<num1<<"\n"<<num2;
}
};

int main()
{   
swAp obj1;
obj1.display();
return 0;
}